src/messages/layouts/MessageLayoutContainer.hpp

```diff
@@ -60,6 +64,7 @@ struct MessageLayoutContainer {
     void breakLine();
     bool atStartOfLine();
     bool fitsInLine(int width_);
+    void reorderRTL(int firstTextIndex);
```

Comment:
Brief comment here how the function achieves its goal, and what `firstTextIndex` is used for